sdk/python/lib/pulumi/dynamic/__main__.py

```diff
@@ -89,7 +100,11 @@ def Diff(self, request, context):
             provider = get_provider(olds)
         else:
             provider = get_provider(news)
-        result = provider.diff(request.id, olds, news)  # pylint: disable=no-member
+        config = self._get_config_arg(provider.diff)
+        if config is not None:
```

Comment:
Does `if config` work here? Ignore me if it does but e.g. we don't do that because linting/style guide/better to be explicit, or if there's a gotcha where it could be falsy but not `None`.

Reply:
`if config` should be fine, not sure why I went with `is not None` :D